pkg/ui/pnpm-lock.yaml

```diff
@@ -39,7 +39,7 @@ overrides:
   '@cockroachlabs/cluster-ui>antd': 3.26.20
   analytics-node>axios-retry: 3.1.9
 
-packageExtensionsChecksum: 39f4d224bf661de76c208d27f33ed125
+packageExtensionsChecksum: c8401d95bcfa63cade05749ae746f3fe
```

Comment:
Hash was different in release-23.1. I reran `pnpm install` and this is the result (still not sure why `webpack-dashboard` and `karma` are popping back in but 🤷